cmd/configrepo/preflight.go

```diff
@@ -48,9 +47,13 @@ func (pr *PreflightRunner) handleApiResponse(res *dub.Response) error {
 		if b, err := ioutil.ReadAll(reader); err != nil {
 			return err
 		} else {
+			if res.IsAuthError() {
+				utils.Die(1, `Invalid credentials. Either the username or password configured is incorrect`)
+			}
+
 			if res.IsError() {
 				if msg, err := api.ParseMessage(b); err == nil {
-					return fmt.Errorf(`Unexpected response: %s`, msg)
```

Comment:
For this particular line, we should just return an `error` as this call is wrapped by `utils.AbortLoudly()`

Reply:
I'll revert this line then